453-skeleton/Texture.cpp

```diff
@@ -0,0 +1,88 @@
+#include "Texture.h"
+#define STB_IMAGE_IMPLEMENTATION
+#include <stb/stb_image.h>
+
+#include <iostream>
+
+Texture::Texture(std::string path, GLint interpolation)
+	: textureID(), path(path), interpolation(interpolation)
```

Comment:
These need to be initialized in the order they are declared in the .h file.

Reply:
Fixed.